mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,28 +1,40 @@
 CLASS net/minecraft/class_1263 net/minecraft/inventory/Inventory
-	METHOD method_18861 countInInv (Lnet/minecraft/class_1792;)I
+	METHOD method_18861 count (Lnet/minecraft/class_1792;)I
+		COMMENT Determines the total number of times the specified item occurs in this inventory across all stored stacks.
 		ARG 1 item
-	METHOD method_18862 containsAnyInInv (Ljava/util/Set;)Z
+	METHOD method_18862 containsAny (Ljava/util/Set;)Z
+		COMMENT Determines whether this inventory contains any of the given candidate items.
 		ARG 1 items
 	METHOD method_5431 markDirty ()V
-	METHOD method_5432 onInvClose (Lnet/minecraft/class_1657;)V
+	METHOD method_5432 onClose (Lnet/minecraft/class_1657;)V
 		ARG 1 player
-	METHOD method_5434 takeInvStack (II)Lnet/minecraft/class_1799;
+	METHOD method_5434 removeStack (II)Lnet/minecraft/class_1799;
+		COMMENT Removes a specific number of items from the given slot.
+		COMMENT
+		COMMENT @returns the removed items as a stack
 		ARG 1 slot
 		ARG 2 amount
-	METHOD method_5435 onInvOpen (Lnet/minecraft/class_1657;)V
+	METHOD method_5435 onOpen (Lnet/minecraft/class_1657;)V
```

Comment:
When used on a BlockEntity for example it may not be clear what some of these methods mean.

 `onOpen` what opened? 

`isValid` what is valid?

In the sense of a Inventory on its own this may make sense, but not where its commonly used.

Reply:
Since `Inventory` should actually be `Container`, `onOpen` makes sense here. For example, `ChestBlockEntity.onOpen` means the chest (container) is opened.